crypto/package.json

```diff
@@ -16,11 +16,10 @@
     "@zk-kit/baby-jubjub": "^0.1.1",
     "@zk-kit/eddsa-poseidon": "^0.5.1",
     "@zk-kit/poseidon-cipher": "^0.1.1",
-    "ethers": "^5.4.7",
+    "ethers": "^6.9.2",
     "optimisedmt": "^0.0.9"
```

Comment:
This PR also included ethers v5 to v6 upgrade?

Reply:
Yes, after that it will be the same version across the whole project. Only crypto package didn't have v6.